OpenTelemetry .NET Automatic Instrumentation v1.5.0 (#4272)
open-telemetry/opentelemetry.io: content/en/docs/languages/net/automatic/_index.md

## Compatibility

> OpenTelemetry .NET Automatic Instrumentation attempts to work with all
> officially supported operating systems and versions of

Reviewer: ```suggestion
OpenTelemetry .NET Automatic Instrumentation should work with all
officially supported operating systems and versions of
```
